### DISCUSSION TITLE
Are new AMD Zen 4 CPUs supported, as they support AXV-512?

### DISCUSSION THREAD

**Discussion by knuser** [milvus-io/milvus#20133] [Q&A and General discussion]
According to Hardware requirements https://milvus.io/docs/v2.1.x/prerequisite-docker.md, required CPU instruction set are:
* SSE4.2
* AVX
* AVX2
* AVX-512

Now it seems that Zen 4 CPUs have AVX-512 support (and first three instruction sets are also supported from previous generations).

Does this mean that Zen 4 CPUs are now fully supported by Milvus 2.1? Or are there other things, that disqualify AMD CPUs?

**Comment by yhmo** [comment]
Please take a look this comment: https://github.com/milvus-io/milvus/discussions/19203#discussioncomment-3643205
